src/components/PineTypes/Enum.js

```diff
@@ -0,0 +1,28 @@
+import { h } from 'preact'
+import Select from '../Select'
+import _ from 'lodash'
+
+export const Edit = ({ schema, value, onChange, ...props }) => (
+  <Select {...props} value={value} onChange={e => onChange(e.target.value)}>
+    {_.map(schema.values, item => {
+      if (_.isString(item)) {
+        return <option>{item}</option>
```

Comment:
I guess that we don't need something like `value={item}`, do we?

Reply:
`<option>` tags default to using their inner text as the value. If you need to provide a different label to the value then you can pass in an object here.